src/test/groovy/graphql/util/AnonymizerTest.groovy

```diff
@@ -723,39 +723,39 @@ type Object1 {
 
         then:
         newSchema == """\
```

Comment:
stripIndent behaves differently on Java 16+ because it overshadows the Groovy method that was present on String before that.
By just dedenting the tests it works

Also the case for the other two tests